src/index.tsx

```diff
@@ -45,8 +46,9 @@ Sentry.init({
   },
 });
 
-const sendToSentry = (error: Error, info: { componentStack: string }) => {
-  Sentry.captureException(error, {
+const sendToSentry = (error: Error, info: ErrorInfo) => {
+  Sentry.captureMessage(`Kritieke applicatie fout: ${error.message}`, {
+    level: 'fatal',
```

Comment:
what's the purpose of this?

Reply:
https://docs.sentry.io/platforms/javascript/usage/set-level/ maar wat het doet in de Sentry omgeving moet ik nog even ondervinden :-)